ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -180,25 +182,33 @@ public struct AvatarView: View {
                         defaultAccessibilityText)
         }()
 
-        let avatarContent = (avatarImage != nil) ?
-            AnyView(Image(uiImage: avatarImage!)
-                        .resizable()
-                        .foregroundColor(Color(foregroundColor)))
-            :
-            AnyView(Text(initialsString)
-                        .foregroundColor(Color(foregroundColor))
-                        .font(Font(tokens.textFont))
-                        .animation(.none))
-
-        let bodyView = tokens.style == .group ?
-        AnyView(avatarContent
+        @ViewBuilder
+        var avatarContent: some View {
+            if let image = avatarImage {
+                Image(uiImage: image)
+                    .resizable()
+                    .foregroundColor(Color(foregroundColor))
+            } else {
+                Text(initialsString)
+                    .foregroundColor(Color(foregroundColor))
+                    .font(Font(tokens.textFont))
+                    .animation(.none)
+            }
+        }
+
+        @ViewBuilder
+        var avatarBody: some View {
+            if tokens.style == .group {
+                avatarContent
                     .background(Rectangle()
-                                    .frame(width: tokens.avatarSize, height: tokens.avatarSize, alignment: .center)
-                                    .foregroundColor(Color(backgroundColor)))
-                    .frame(width: tokens.avatarSize, height: tokens.avatarSize, alignment: .center)
-                    .cornerRadius(tokens.borderRadius))
-            :
-        AnyView(Circle()
+                                        .frame(width: tokens.avatarSize, height: tokens.avatarSize, alignment: .center)
+                                        .foregroundColor(Color(backgroundColor)))
+                        .frame(width: tokens.avatarSize, height: tokens.avatarSize, alignment: .center)
+                        .contentShape(RoundedRectangle(cornerRadius: tokens.borderRadius))
+                        .clipShape(RoundedRectangle(cornerRadius: tokens.borderRadius))
+                        .cornerRadius(tokens.borderRadius)
```

Comment:
Do you need the clip shape and the corner radius? Seems like the clip shape should take care of rounding the corners for you.

Reply:
You're right. Just the contentShape and clipShape are enough to do the trick.